src/app/content/components/Toolbar/index.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import ReactType from 'react';
+import { Provider } from 'react-redux';
+import rendererType, { act } from 'react-test-renderer';
+import createTestStore from '../../../../test/createTestStore';
+import { receiveUser } from '../../../auth/actions';
+import { User } from '../../../auth/types';
+import MessageProvider from '../../../MessageProvider';
+import { Store } from '../../../types';
+import { closeMyHighlights, openMyHighlights } from '../../highlights/actions';
+import HighlightButton from './HighlightButton';
+import HighlightsPopUp from './HighlightsPopUp';
+
+describe('MyHighlights', () => {
```

Comment:
this file has tests for both the highlights button and the modal popup, neither of those tests really have anything to do with the toolbar but the test file is called `Toolbar/index.spec.tsx`.

i think these tests look good, but they should be separated out for the components that they're testing.